packages/affiliate/tests/php/test_Affiliate.php

```diff
@@ -0,0 +1,94 @@
+<?php
+namespace Automattic\Jetpack\Partners;
+
+use Automattic\Jetpack\Constants as Jetpack_Constants;
+use phpmock\Mock;
+use phpmock\MockBuilder;
+use PHPUnit\Framework\TestCase;
+
+class Test_Affiliate extends TestCase {
+
+	/**
+	 * Mock affiliate code.
+	 *
+	 * @var string
+	 */
+	private $aff_code = 'abc123';
+
+	/**
+	 * Test teardown.
+	 */
+	public function tearDown() {
+		Mock::disableAll();
+	}
+
+	function test_affiliate_code_missing() {
+		$this->mock_function_with_args( 'get_option', [
+			[
+				'jetpack_affiliate_code',
+				'',
+				''
+			]
+		] );
+
+		$this->mock_function_with_args( 'apply_filters', [
```

Comment:
It appears that we're doing this one for all tests, so I wonder if we could move it to a helper method.